application/modules.go

```diff
@@ -63,4 +64,5 @@ var ModuleBasics = module.NewBasicManager(
 	liquidity.AppModuleBasic{},
 	router.AppModuleBasic{},
 	ica.AppModuleBasic{},
+	wasm.AppModuleBasic{},
```

Comment:
do not add wasm module here.. no need for that. that wiring will be done in the persistenceCore